src/libcore/cmp.rs

```diff
@@ -880,24 +880,24 @@ mod impls {
 
     ord_impl! { char usize u8 u16 u32 u64 u128 isize i8 i16 i32 i64 i128 }
 
-    #[unstable(feature = "never_type", issue = "35121")]
+    #[stable(feature = "never_type", since = "1.24.0")]
```

Comment:
Shouldn't this be `since = "1.25.0"`?

Reply:
True, it's too late to make 1.24. Thanks for the catch!